extensions/wikia/CommunityPage/helpers/LinkHelper.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+class LinkHelper {
+	public static function forceLoginLink(Title $title, $edit=true ) {
+		global $wgUser, $wgDisableAnonymousEditing;
+
+		if ( $wgDisableAnonymousEditing && !$wgUser->isLoggedIn() ) {
+			return SpecialPage::getTitleFor( 'SignUp' )->getFullURL(
+				$query=[
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$query" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)